src/oni2-keyboard-layout/Oni2_KeyboardLayout.rei

```diff
@@ -0,0 +1,23 @@
+type callback = unit => unit;
+
+let init: unit => unit;
+let subscribe: (callback, unit) => unit;
+let getCurrentLanguage: unit => string;
+let getCurrentLayout: unit => string;
+
+module Keymap: {
+  type entry = {
+    unmodified: option(string),
+    withShift: option(string),
+    withAltGraph: option(string),
+    withAltGraphShift: option(string),
+  };
+
+  type t;
+
+  let getCurrent: unit => t;
+
+  let entryOfScancode: (t, Sdl2.Scancode.t) => option(entry);
```

Comment:
Awesome! This looks like it'll be easy to map from Sdl2 keys -> keymaps now

Reply:
I just added a usage of the library in `InputStoreConnector` as a proof-of-concept - right now, it just logs the keys, but this is where I'll need to resolve the scancode to improve matching.